web.test.tests/web.test.tests/Tests/DepositePageTests.cs

```diff
@@ -20,16 +19,20 @@ class DepositePageTests
         public void Setup()
         {
             driver = new ChromeDriver();
-
             driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
             driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
-
             driver.Url = "http://localhost:64177/Login";
 
-            driver.FindElement(By.Id("login")).SendKeys("test");
-            driver.FindElement(By.Id("password")).SendKeys("newyork1");
-            driver.FindElement(By.Id("loginBtn")).Click();
+            new LoginPage(driver).Login("test", "newyork1");
+            new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
```

Comment:
I suggest moving this as a new method to LoginPage:

```
public DepositePage Login()
{
    Login("test", "newyork1");
    new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
    return new DepositePage(driver);
}
```

```suggestion
            new LoginPage(driver).Login();
```

Reply:
Done